### PR TITLE
Disabled

### PR DESCRIPTION
**Pull request by junca8**

### CODE REVIEW

File: src/select.js [hunk 1]
```diff
@@ -216,7 +216,22 @@ export default class CanopySelect extends React.Component {
         if (option.separator) {
           return <li key={`separator${index}`} className="separator" />
         } else {
-          return <li key={option.key} className={selectedIndex === index ? '+selected' : ''} onMouseDown={this.selectItem.bind(this, index)}><a style={option.value !== null ? {} : {color: "rgba(0,0,0,0)"}}>{option.value !== null ? option.value : "null"}</a></li>
+          const linkStyles = {
+            color: option.value !== null ? "" : "rgba(0,0,0,0)",
```
Comment (geoctrl) [inline src/select.js]: what's `rgba(0,0,0,0)` for? that would be black with `0` alpha

Comment (geoctrl) [reply src/select.js]: you could also just say `transparent`

Comment (junca8) [reply src/select.js]: that was part of the existing code. not sure why it was written that way.

Comment (geoctrl) [reply src/select.js]: 🤷‍♂

**Pull request merged**